### PR TITLE
Hello Microverse Branch

### PR DESCRIPTION
**Pull request by Geoelycom**

### CODE REVIEW

**Comment by Davidosky007** [comment]
# Hi @Geoelycom, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️
* your pull request should have a descriptive title telling what changes are inside and also it could have a short summary describing introduced changes in more details. these make reviewing easy.
* you should consider setting up your linters properly by adding the necessary files and folders as stated in the project requirement.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL]
You can also consider:
- I suggest you begin your commit messages with capital letters as this is a best practice 👍 

Cheers and Happy coding!👏👏👏

# Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Geoelycom** [comment]
I have made the requested changes you asked for.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Geoelycom!  

You worked hard in all the instructions that the previous Code Reviewer gives you👍🏽 

[OPTIONAL] 

- [ ] [OPTIONAL] Your Pull Request title must be descriptive and give us a general understanding of the project, not only the name of the branch (e.g. "First set up" or "HelloMicroverse project's set up"; and you must fill in a descriptive message about what changes you did to your project. This is a good practice, please create a descriptive message.

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/test.md
```
[the remark this answers is not in the document]
Comment (Geoelycom) [reply .gitignore]: thank you very much.

**Pull request merged**